### PR TITLE
LPA-2204 added phpunit container

### PR DESCRIPTION
**Pull request by m3dwards**

### CODE REVIEW

File: phpunit/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,17 @@
+FROM registry.service.opg.digital/opguk/php-fpm
+
+WORKDIR /tmp
+
+ADD  docker/phpunit-4.8.35.phar phpunit-4.8.35.phar
```
Comment (carinadigital) [inline phpunit/Dockerfile]: Why not add directly to /usr/local/bin/phpunit

File: phpunit/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+RUN chmod +x phpunit-4.8.35.phar
+RUN mv phpunit-4.8.35.phar /usr/local/bin/phpunit
+
+RUN apt-get install php5-mcrypt
```
Comment (carinadigital) [inline phpunit/Dockerfile]: The newer opguk/php-fpm containers should include php5-mcrypt. Can you check please?

File: phpunit/Dockerfile (hunk 1 above)
Comment (m3dwards) [reply phpunit/Dockerfile]: agreed

File: phpunit/Dockerfile (hunk 2 above)
Comment (m3dwards) [reply phpunit/Dockerfile]: will check

Comment (m3dwards) [reply phpunit/Dockerfile]: it does

**Pull request merged**